assistant/src/main/java/com/ibm/watson/assistant/v1/model/DialogNodeOutputGenericDialogNodeOutputResponseTypeConnectToAgent.java

```diff
@@ -20,7 +20,7 @@
 public class DialogNodeOutputGenericDialogNodeOutputResponseTypeConnectToAgent
     extends DialogNodeOutputGeneric {
 
-@SerializedName("transfer_info")
+  @SerializedName("transfer_info")
```

Comment:
Formatting change?

Reply:
Yes, not sure why the formatting was wrong initially but the spacing looks correct here